### ISSUE TITLE
Move perf metrics into performance section

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#1876]
once **Add TTI < 10s audit for PWA** #1840 is done, we can move perf metrics to their real home, out of the PWA section.

**Comment by addyosmani** [comment]
Do we have a public note about why we're extending TTI's budget to < 10s? (just for historical context as folks look at how/why metric budgets have evolved).

**Comment by paulirish** [comment]
@addyosmani there's two separate "audits" here

A) A boolean audit saying "is the TTI faster than 10s?". This is to mirror this item in the baseline checklist: 
![image](https://cloud.githubusercontent.com/assets/39191/24376450/c24637a6-12f0-11e7-8a5f-363349fef422.png)

B) sharing the value and scoring of TTI, along with the other key perf metrics. The scoring over here is separate and we can handle completely independently from the PWA requirement. FYI: With our current scoring curve (which honestly hasn't been validated), a 9900ms TTI gets a score of 11%. That seems a tad low, but I'm also okay with Lighthouse perf section having much higher standards for perf than the hard boolean requirements of PWA-ness.

**Comment by patrickhulce** [comment]
This is already done in the categories landing of config v2. OK to close @paulirish or was there more to it (UI perhaps?) than just moving them out?

**Comment by paulirish** [comment]
yah sgtm.
